### PR TITLE
Added styling over TCP, updated settings tab info

### PR DESCRIPTION
**Pull request by Haavard15**
Amateur programmer and first actual pull request.

I needed to be able to style buttons over TCP so I added changing text and color/bgcolor.

I also noticed that the HTTP API had some un-documented features so I added those to the settings tab along the new things.

Let me know if there is anything I should change.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -548,6 +554,37 @@ <h4>TCP/UDP Remote control</h4>
 										<p>Press page 1 bank 2<br>
 											<code>BANK-PRESS 1 2</code></p>
 
+										<h4>HTTP Remote control</h4>
+										<p>Remote triggering can be done by sending HTTP Requests to the same IP and port Companion is running on.</p>
+										<p><strong>Commands:</strong>
+											<ul>
+												<li><code>/press/bank/</code> &lt;page&gt;<code>/</code>&lt;bank&gt;<br>
+												<i>Press and release a button (run both down and up actions)</i></li>
+												<li><code>/style/bgcolor/</code>&lt;page&gt;<code>/</code>&lt;bank&gt;<code>?text=</code>&lt;bgcolor HEX&gt;<br>
```
Comment (Julusian) [inline public/index.html]: This does not look correct to me.
from a skim of the code it looks like it should be `/style/bank/<page>/<bank>?bgcolor=<hex>` 
etc

Comment (Haavard15) [reply public/index.html]: Good catch! I committed to my own repo, does it look correct now or do I have to do anything more to make it appear here?

File: public/index.html [hunk 2]
```diff
@@ -548,6 +554,37 @@ <h4>TCP/UDP Remote control</h4>
...
+												<i>Change text size on a button (between the predefined values)</i></li>
+											</ul>
+										</p>
+
+										<p>
+											<strong>Examples</strong>
+										</p>
+
+										<p>Press page 1 bank 2<br>
+											<code>/style/bank/1/2</code></p>
```
Comment (Julusian) [inline public/index.html]: should be `/press/bank/1/2`

File: public/index.html (hunk 1 above)
Comment (Julusian) [reply public/index.html]: It shows up here too :)

File: public/index.html (hunk 2 above)
Comment (Haavard15) [reply public/index.html]: Thanks :-)

**Pull request merged**